lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -343,6 +343,17 @@ export class BrowserStorage extends CacheManager {
         this.setItem(this.generateCacheKey(authorityKey), authority, CacheSchemaType.TEMPORARY);
     }
 
+    /**
+     * Gets the cached authority based on the cached state. Returns empty if no cached state found.
+     */
+    getCachedAuthority(): string {
+        const state = this.getItem(this.generateCacheKey(TemporaryCacheKeys.REQUEST_STATE), CacheSchemaType.TEMPORARY) as string;
+        if (!state) {
+            return "";
```

Comment:
I think `null` is a better semantic representation of not being found that empty string.

Reply:
done.